aptos-move/move-examples/hello_blockchain/sources/hello_blockchain.move

```diff
@@ -47,6 +47,11 @@ module hello_blockchain::message {
 
     #[test(account = @0x1)]
     public entry fun sender_can_set_message(account: signer) acquires MessageHolder {
+        use std::debug;
```

Comment:
Usually you put these outside of the function.  Did you test this?

Reply:
Yes i tested it works. For some reason VSCode was complaining about unused imports when I imported at the top of the module so I put it here instead... but I can move it to the top of the module if that's better

Reply:
Yeah you have to put a #[test_only] annotation, as it's unused outside of tests

Reply:
Done!